recipes/behst/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+mkdir -p $PREFIX/bin
+chmod +x bin/*
+cp bin/* $PREFIX/bin
```

Comment:
@davidechicco note no downloading of data in build.sh. Just copying executables to `$PREFIX/bin`